problem_builder/mentoring.py

```diff
@@ -322,6 +329,40 @@ class MentoringBlock(BaseMentoringBlock, StudioContainerXBlockMixin, StepParentM
     )
 
     @property
+    def allowed_nested_blocks(self):
+        """
+        Returns a list of allowed nested XBlocks. Each item can be either
+        * An XBlock class
+        * A NestedXBlockSpec
+
+        If XBlock class is used it is assumed that this XBlock is enabled and allows multiple instances.
+        NestedXBlockSpec allows explicitly setting disabled/enabled state, disabled reason (if any) and single/multiple
+        instances
+        """
+        additional_blocks = []
+        try:
+            from xmodule.video_module.video_module import VideoDescriptor
+            additional_blocks.append(NestedXBlockSpec(
+                VideoDescriptor, category='video', label=_(u"Video")
+            ))
+        except ImportError:
+            pass
+        try:
+            from imagemodal import ImageModal
+            additional_blocks.append(NestedXBlockSpec(
+                ImageModal, category='imagemodal', label=_(u"Image Modal")
+            ))
+        except ImportError:
+            pass
+
+        return [
+            NestedXBlockSpec(AnswerBlock, boilerplate='studio_default'),
+            MCQBlock, RatingBlock, MRQBlock,
+            NestedXBlockSpec(None, category="html", label=self._("HTML")),
+            AnswerRecapBlock, MentoringTableBlock, PlotBlock, SliderBlock
```

Comment:
It looks like there are three types of children missing from this list: "Message (Complete)", "Message (Incomplete)", and "Message (Max # Attempts)" (see screenshot). It's not visible in the screenshot, but I think there is also "Message (Assessment Review)" when using the deprecated assessment mode.

![screen shot 2016-09-29 at 10 04 22](https://cloud.githubusercontent.com/assets/32585/18946153/edf15f98-862d-11e6-8faf-46dbfac3ccc8.png)